src/components/Title/Title.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface TitleProps {
+  children: React.ReactNode
+}
+
+export const Title = ({ children }: TitleProps): React.ReactElement => {
```

Comment:
for the sake of flexibility, I think we should try to extend the React HTML attribute props wherever possible, and that way someone can pass in standard React props such as `className` & `children`, as well as native HTML attributes (like aria attributes) without us having to explicitly handle them. I need to add documentation to this repo about taking that approach because I don't think it is well known, but it would look something like this:

```
export const Title = ({ className, children, ...props } : TitleProps & React.HTMLAttributes<HTMLDivElement>): React.ReactElement => {

...

return (
  <div className={classes} {...props}>
  ...
```